string transform_to
perception/Area area
perception/FilterOptions filter_option
---
perception/ROI[] rois